### DISCUSSION TITLE
How soon the XCode 13.x will be added?

### DISCUSSION THREAD

**Discussion by lzaprivoda** [actions/virtual-environments#4416] [Installed software]
Hi! I need an environment with XCode 13 and a new building system for my project Fastlane builds. 
How soon will it be available?
Thanks!

**Comment by miketimofeev** [comment]
@lzaprivoda Hi!
It's already available on the macOS-11 image:
https://github.com/actions/virtual-environments/blob/main/images/macos/macos-11-Readme.md#xcode
